weekly-updates/project-lead-meetings/10_14_18.md

```diff
@@ -89,13 +89,13 @@ _TO DO (for all of next semester)_:
 
 ## sledge
 
-_UPDATES_:
+_UPDATES_: The Good: It worked, ratings updated as expected, everyone got it running (with some refreshes) The Bad: Not enough judges on the top prizes, lots of very close scores, The Ugly: Table numbers didn't work
 
-_QUESTIONS_:
+_QUESTIONS_: What's a better system for choosing between close scores?
 
-_TO DO (for all of next semester)_:
+_TO DO (for all of next semester)_: General improvements, table system and better tie-breaking capabilities (maybe add more judges to top hacks)
 
-_What did the Devpost guy throw on just before judging? Was it necessary? When did he approach you?_:
+_What did the Devpost guy throw on just before judging? Was it necessary? When did he approach you?_: We were originally going to post a Google Sheets form of all the hacks and their table numbers. MLH said that wasn't good enough since it would require sponsors to manually look up table numbers for hacks running for their prize. Devpost guy said he had an app that could import the table numbers and automatically associate everything with the devpost data. Presumably due to a "Test Hack" leftover from the demo some table numbers didn't register correctly, and Sledge didn't match the numbers. The import happened about 15 minutes before judging, and the problem was noticed about 10 minutes into judging.
```

Comment:
OK.

For future reference: we plan on forcing all software used by non-hackers be known to us a month or so out to avoid any stress from debugging their shit.

Can we add listing by prize as a feature?

Reply:
List by prize will definitely be there by next hackru